Add removed info for kyma-developer role (#8106)
kyma-project/kyma: docs/security/03-05-roles-in-kyma.md

| **kyma-admin** | The role with the highest permission level which gives access to all Kubernetes and Kyma resources and components with administrative rights. |
| **kyma-namespace-admin** | The role that has the same rights as the **kyma-admin** role, except for the write access to [AddonsConfigurations](https://kyma-project.io/docs/master/components/helm-broker#custom-resource-addons-configuration). |

> >**CAUTION:** To give a user the **kyma-developer** role permissions in a Namespace, create a RoleBinding to the **kyma-developer** cluster role in that Namespace. You can specify as a subject of the RoleBinding User using email or Group.

Reviewer: ```suggestion
>**CAUTION:** To give a user the **kyma-developer** role permissions in a Namespace, create a RoleBinding to the **kyma-developer** cluster role in that Namespace. You can define a subject of the RoleBinding by specifying either a **User Group**, or a **User** by means of a user email. 
```
